src/Pie.js

```diff
@@ -20,50 +20,127 @@ function createPath(cx, cy, r, startAngle, arcAngle) {
       startAngle + arcAngle,
     );
   } else {
-    p.path.push(0, cx + r * Math.cos(startAngle), cy + r * Math.sin(startAngle));
-    p.path.push(4, cx, cy, r, startAngle, startAngle + arcAngle, 1);
+  //starting point of our chart
+  if(isBezian){
+    let ROUNDNESSOUSIDE = 1 - ((r - innerRadius)/ innerRadius) - (arcAngle * .5);
+    let ROUNDNESSINSIDE = 1 + ((r - innerRadius)/ innerRadius) + (arcAngle * .5);
+    const PULLBACK = 0.05;
+    const ANCHORFORWARD = .2 ;
+      //This is for the part that is the divider
+      p.moveTo(cx + r * ROUNDNESSOUSIDE * Math.cos(startAngle + PULLBACK), cy + r * ROUNDNESSOUSIDE * Math.sin(startAngle + PULLBACK));
+      p.onBezierCurve(
+        undefined,
+        undefined,
+        cx + r  * ROUNDNESSOUSIDE * Math.cos(startAngle + PULLBACK),
+        cy + r  * ROUNDNESSOUSIDE * Math.sin(startAngle + PULLBACK),
+        cx + r * Math.cos((startAngle + ANCHORFORWARD)),
+        cy + r * Math.sin((startAngle + ANCHORFORWARD)),
+        cx + r * ROUNDNESSINSIDE * Math.cos(startAngle + PULLBACK),
+        cy + r  * ROUNDNESSINSIDE * Math.sin(startAngle + PULLBACK),
+      );
+    }else{
+      //This is for the main arc of the pie chart
+      p.moveTo(cx + r * Math.cos(startAngle), cy + r * Math.sin(startAngle));
+      p.onArc(
+        undefined,
+        undefined,
+        undefined,
+        undefined,
+        cx,
+        cy,
+        r,
+        r,
+        startAngle,
+        startAngle + arcAngle,
```

Comment:
This is. exactly same as what we have inside first if condition `if (Platform.OS === 'web') {`.

Reply:
@nihgwu I'm not sure why this ``` if (Platform.OS === 'web') { ``` is here when we should never be in web I just changed it so I wouldn't remove the logic

Reply:
I thought you have remove the web support before?